ayup/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,25 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> facts;
+
+  @Override
+  public void init() {
```

Comment:
You might be able to get away with this fact list being a static private ArrayList?

Reply:
+1. I don't think this is the intended use of the `init()` method. https://www.gnu.org/software/classpathx/servletapi/javadoc/javax/servlet/GenericServlet.html

Reply:
So where do I add the facts to the fact list?

Reply:
Usually like this.
```java
private static final ImmutableList<String> FUN_FACTS =
  ImmutableList.of(
      "fact 1", "fact 2", ... );
```

Reply:
I see, thanks!